Dockerfile.ubuntu

```diff
@@ -260,6 +260,7 @@ RUN if [ "$NVIDIA" == "1" ] ; then true ; else exit 0 ; fi ; \
 WORKDIR /ovms
 # GPU testing in build img
 COPY ./install_ubuntu_gpu_drivers.sh /tmp/install_gpu_drivers.sh
+ARG INSTALL_DRIVER_VERSION="23.22.26516"
```

Comment:
Add for redhat?

Reply:
There was no issue for redhat, `ARG INSTALL_DRIVER_VERSION` is added **before** running the installation script.

Reply:
Later on ARG INSTALL_DRIVER_VERSION="24.26.30049" is set (line 407)

Reply:
yes, and it should be since it is multistage and line 407 is in different stage (which do not share args) @ngrozae